### PR TITLE
gh-2828 Adjust face id unlock screen layout

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #2828

Changes proposed in this pull request:
- Remove toolbar underline
- Adjust content color

### CODE REVIEW

File: Multisig/UI/App/Unlock/FaceIDUnlockViewController.xib [hunk 1]
```diff
@@ -45,7 +45,7 @@
                 </label>
             </subviews>
             <viewLayoutGuide key="safeArea" id="fnl-2z-Ty3"/>
-            <color key="backgroundColor" name="backgroundPrimary"/>
+            <color key="backgroundColor" name="backgroundUnlock"/>
```
Comment (MouazAlzahabi) [inline Multisig/UI/App/Unlock/FaceIDUnlockViewController.xib]: can't we just go with backgroundPrimary ?

Comment (elgatovital) [reply Multisig/UI/App/Unlock/FaceIDUnlockViewController.xib]: backgroundPrimary is not completely white in day mode and not completely black in night mode, so you will see the difference in color between toolbar and content area.

Comment (MouazAlzahabi) [reply Multisig/UI/App/Unlock/FaceIDUnlockViewController.xib]: Maybe it's because wrong constraints ?

Comment (elgatovital) [reply Multisig/UI/App/Unlock/FaceIDUnlockViewController.xib]: no, it's the color issue

**Comment by MouazAlzahabi** [comment]
@elgatovital this is how the screen looks right now 
IMG_90F7D9392C5B-1

**Review by MouazAlzahabi — APPROVED** [review]

**Pull request merged**